Remotion/Core/Collections.Caching.UnitTests/LockingCacheDecoratorTest.cs

```diff
@@ -108,8 +108,7 @@ public void GetOrCreateValue_WithNestedGetOrCreatedValue_ThrowsInvalidOperationE
 
       Assert.That (actualValue, Is.EqualTo (expected));
 
-      object actualValue2;
-      Assert.That (cache.TryGetValue ("key1", out actualValue2), Is.True);
+      Assert.That (cache.TryGetValue ("key1", out var actualValue2), Is.True);

```

Comment:
Same question as above - `out var` vs declaration. Multiple occurrences in this file.